packages/curve-ui-kit/src/widgets/Footer/Link.tsx

```diff
@@ -0,0 +1,56 @@
+import LinkMui from '@mui/material/Link'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { ReactNode } from 'react'
+import { SizesAndSpaces } from 'curve-ui-kit/src/themes/design/1_sizes_spaces'
+import { Fonts } from 'curve-ui-kit/src/themes/typography'
+
+export type LinkProps = {
+  label: string
+  href: string
+  icon?: ReactNode
+}
+
+export const Link = ({ label, href, icon }: LinkProps) => (
+  <LinkMui
+    href={href}
+    target="_blank"
+    rel="noreferrer"
+    underline="hover"
+    sx={{
+      display: 'grid',
+      gridTemplateColumns: '16px 1fr',
+      gap: SizesAndSpaces.Spacing.sm,
+      alignItems: 'center',
+    }}
+  >
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        '& svg': {
+          width: '100%',
+          height: '100%',
+        },
+      }}
+    >
+      {icon ? icon : null}
+    </Box>
+
+    <Typography
+      variant="ghost"
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        height: '1.5rem',
+        fontFamily: Fonts['Mona Sans'],
+        fontSize: SizesAndSpaces.FontSize.md,
+        // In dark mode the bolder font is very hard to read.
+        fontWeight: (t) =>
+          t.palette.mode === 'dark' ? SizesAndSpaces.FontWeight.Normal : SizesAndSpaces.FontWeight.Medium,
+      }}
+    >
+      {label}
+    </Typography>
+  </LinkMui>
```

Comment:
I think we can just use the mui button with ghost color and the startIcon prop

```suggestion
  <Button
    component={MuiLink}
    href={href}
    target="_blank"
    rel="noreferrer"
    size="small"
    color="ghost"
    startIcon={icon}
  >
      {label}
  </Button>
```

Reply:
I gave it a try but it opens a new can of worms with respect to default button style and behavior like hovers that make it look nothing like the Figma reference. Undoing that styling wouldn't make it any easier, so I've resorted back to using a MUI Link. I have however been able to remove some styling.

https://github.com/user-attachments/assets/dfed90fc-6f9c-4796-bfb5-7fb50493895d